### DISCUSSION TITLE
Questions about kernels

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#23379] [Q&A Modules: Phase field] [answered]
Hello MOOSE experts,
I am trying to simulate fracture process using phase field module. But one needed kernel "PhaseFieldFractureMechanicsOffDiag" is not documented yet. Would someone please give me some explanation what does it do? Another question is that the nonconserved parameters evolution equation is decomposed into 3 parts and compute residuals: TimeDerivative, AllenCahn, and ACInterface, are these residuals summed up during solving?

**Comment by GiudGiud** [comment, marked as answer]
@sveerara
